mne/preprocessing/infomax_.py

```diff
@@ -106,8 +106,8 @@ def infomax(data, weights=None, l_rate=None, block=None, w_change=1e-12,
     signsbias = 0.02
     signcount_threshold = 25
     signcount_step = 2
-    if ext_blocks > 0:  # allow not to recompute kurtosis
-        n_subgauss = 1  # but initialize n_subgauss to 1 if you recompute
+    # if ext_blocks > 0:  # allow not to recompute kurtosis
+    #     n_subgauss = 1  # but initialize n_subgauss to 1 if you recompute
```

Comment:
good question. Maybe @jmontoyam can tell whi this code is unused as he recenty parsed the code in detail.